pages/docs/data-ops.mdx

```diff
@@ -39,14 +39,14 @@ Hashboard allows you to represent your resources as configuration files, which c
   <Card
     icon={<Terminal {...iconProps} />}
     title="Explore Hashboard commands"
-    href="/docs/data-ops/cli"
+    href="/docs/data-ops/cli/v3"
   >
     Master Hashboard's CLI with these essential commands
   </Card>
   <Card
     icon={<CheckCircle {...iconProps} />}
     title="Validate and preview updates"
```

Comment:
The title/description of this card no longer really matches the content in the link, think we should update the build page to talk about previewing a bit more or update this copy